### PR TITLE
[Catalog] Fix catalog info modal text overflow

### LINKED ISSUE #1608: [bug] Fix overflow of text in Catalog Info Modal
### Description

Screenshot 2024-01-25 at 3 24 14 AM

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by dragon-slayer875**
**Description**

This PR fixes #1608 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@dragon-slayer875 the best practice is whenever your raise a pr we should describe the changes

**Comment by sudhanshutech** [comment]
We don't need a extra scroll here @dragon-slayer875 it is also affecting the single catalog page. Instead if you see, there is implementation for section `What this pattern does` where if content is more then a particular character we redirect them to its specific single catalog page to read more, you need to do same with that. Otherwise this scroll not looking good

**Comment by sudhanshutech** [comment]
still working on it @dragon-slayer875 ?

**Comment by saurabh100ni** [comment]
Hi @dragon-slayer875  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

File: _sass/catalog.scss
Comment (sudhanshutech) [inline _sass/catalog.scss:762]: no scroll needed

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@dragon-slayer875 what's the status, need not to take a lot time for a already available css

File: _sass/catalog.scss
Comment (dragon-slayer875) [reply _sass/catalog.scss]: Hello. Apologies for the delay in communication. I have re-adjusted the overflow behavior as requested and have applied the required changes. 
Would request a re-review of the PR. 
Thanks.

**Comment by dragon-slayer875** [comment]
@sudhanshutech is the issue stale?
I was wondering if the PR would be reviewed, so as to move onto other issues.

**Review by vishalvivekm — DISMISSED** [review]
LGTM.
Thank you for your contribution @dragon-slayer875

**Comment by dragon-slayer875** [comment]
@sudhanshutech I have received an approval from a reviewer. I would like to request a re-review for the changes you requested and maybe a merge if the work for this issue is done.

**Comment by dragon-slayer875** [comment]
@sudhanshutech I have resolved the test that was not passing just after the branch merge just now. 
Would like to re-request a review.

**Pull request merged**
